### PR TITLE
Add test suite for sclorg-testing-farm

### PR DESCRIPTION
**Pull request by phracek**
This pull request adds test suite for this repository.

We need to be sure that all required rpms are installed.

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

File: tasks/check_file_presents.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- name: Check if file {{ item }} exists
```
[the remark this answers is not in the document]
Comment (phracek) [reply tasks/check_file_presents.yml:1]: Which one is not? I don't follow.

File: plans/test-plans.fmf [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        # TODO install packages by ansible playbook
+        curl -o /etc/yum.repos.d/distgen-epel-7.repo https://copr.fedorainfracloud.org/coprs/praiskup/distgen/repo/epel-7/praiskup-distgen-epel-7.repo
+        yum -y install podman ansible podman-docker perl git centos-release-scl-rh rsync groff-base centos-release-openshift-origin epel-release http://cbs.centos.org/kojifiles/packages/golang-github-cpuguy83-go-md2man/1.0.4/4.0.el7/x86_64/golang-github-cpuguy83-go-md2man-1.0.4-4.0.el7.x86_64.rpm
+        touch /etc/containers/nodocker
+        yum -y install rh-python36-python-virtualenv origin-clients distgen python3
+        mkdir -p /etc/rhsm/ca && touch /etc/rhsm/ca/redhat-uep.pem
+        git clone $REPO_URL /root/$REPO_NAME
+        cd /root/$REPO_NAME
+        git fetch origin +refs/pull/*:refs/remotes/origin/pr/*
+        git checkout origin/pr/$PR_NUMBER/head
```
[the remark this answers is not in the document]
Comment (phracek) [reply plans/test-plans.fmf]: Currently, this is done in all our plans in prepare section and not in the test.

File: tasks/check_file_presents.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (phracek) [reply tasks/check_file_presents.yml:1]: I got it. You mean RPMs. I will do it as a separate commit.

File: plans/test-plans.fmf (hunk 2 above)
[the remark this answers is not in the document]
Comment (zmiklank) [reply plans/test-plans.fmf]: Yes. And what is the function of it in here?

File: tasks/check_file_presents.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (phracek) [reply tasks/check_file_presents.yml:1]: Fixed by https://github.com/sclorg/sclorg-testing-farm/pull/49/commits/7441ec1ed58fe77378acea1b4a9262289ef104d2

File: plans/test-plans.fmf (hunk 2 above)
[the remark this answers is not in the document]
Comment (phracek) [reply plans/test-plans.fmf]: The file has been renamed.

[the remark this answers is not in the document]
Comment (phracek) [reply plans/test-plans.fmf]: In the case of each PR, we need to be sure, that plan for each OS contains all packages and files.

File: plans/c9s.fmf [hunk 3]
```diff
@@ -2,27 +2,50 @@ summary: TMT/TFT plan for running container tests on CentOS Stream 9
...
+            # TODO install packages by ansible playbook
+            curl -o /etc/yum.repos.d/distgen-epel-9.repo https://copr.fedorainfracloud.org/coprs/praiskup/distgen/repo/epel-9/praiskup-distgen-epel-9.repo
+            yum -y install podman podman-docker git make
+            yum -y install https://kojihub.stream.centos.org/kojifiles/packages/golang-github-cpuguy83-md2man/2.0.0/16.gitaf8da76.el9/x86_64/golang-github-cpuguy83-md2man-2.0.0-16.gitaf8da76.el9.x86_64.rpm
+            touch /etc/containers/nodocker
+            git clone $REPO_URL /tmp/$REPO_NAME
+            cd /tmp/$REPO_NAME
+            git fetch origin +refs/pull/*:refs/remotes/origin/pr/*
+            git checkout origin/pr/$PR_NUMBER/head
+            git submodule update --init
```
Comment (zmiklank) [inline plans/c9s.fmf:17]: Is the indentation correct here?

File: plans/centos7.fmf [hunk 4]
```diff
@@ -27,3 +19,33 @@ prepare:
...
+        how: shell
+        tests:
+        - name: Run CentOS7 tests on CentOS-7, depends on TEST_NAME
+          framework: shell
+          test: cd /root/$REPO_NAME && make $TEST_NAME TARGET=$OS SINGLE_VERSION=$SINGLE_VERSION
+          duration: 3h
+
+/test-plan:
+    adjust:
+      - because: "Plan to be ran when either executed locally, or executed by CI system."
```
Comment (zmiklank) [inline plans/centos7.fmf:40]: Could the reason why we have 2 discover sections be described here?
Now the description does not say much.

**Review by zmiklank — CHANGES_REQUESTED** [review]
Thank you for the PR, this looks great.
I have met only small issues (see in comments).

File: plans/c9s.fmf (hunk 3 above)
Comment (phracek) [reply plans/c9s.fmf]: Fixed by commit https://github.com/sclorg/sclorg-testing-farm/pull/49/commits/601b457666a41779c6a503838fc7ce84c769d829

File: plans/centos7.fmf (hunk 4 above)
Comment (phracek) [reply plans/centos7.fmf]: Fixed by commit https://github.com/sclorg/sclorg-testing-farm/pull/49/commits/8c907109cb71797abb70ec1bdce20504971a73d7

**Review by zmiklank — APPROVED** [review]

**Pull request merged**